middle_end/flambda2/to_cmm/to_cmm_shared.mli

```diff
@@ -37,14 +37,17 @@ val symbol_from_linkage_name :
 
 val symbol : dbg:Debuginfo.t -> Symbol.t -> Cmm.expression
 
+(** This does not inline effectful expressions. *)
 val name :
   To_cmm_env.t ->
   Name.t ->
   Cmm.expression * To_cmm_env.t * Effects_and_coeffects.t
 
 val const : dbg:Debuginfo.t -> Reg_width_const.t -> Cmm.expression
 
+(** [consider_inlining_effectful_expressions] defaults to [false]. *)
```

Comment:
The default behaviour is to use `Flambda_features.Expert.inline_effects_in_cmm`, which itself defaults to false if no flag is given. Maybe the comment could reflect that in some way ?

Reply:
Fixed